docs/user-guide-web.md

```diff
@@ -0,0 +1,105 @@
+# Cyclone Web User Guide  
+  
+`Cyclone Web` provides a user friendly interface for users to interact with Cyclone. This document guides users how to use Cyclone web to build and execute workflows.
+
+## Introduction
+
+Cyclone Web consists of five components:
+
+- Overview: Overview of the Cyclone platform
+- Projects: Workflows are managed in projects, it's the main component of Cyclone web
+- Templates: Stages templates for Cyclone
+- Integrations: Integrated external services like Github, Docker registry.
+- Resource Types: Manage resource types Cyclone supported
+
+### Overview
+
+![overview](./images/screenshots/overview.png)
+
+### Resource Types
+
+Resource in Cyclone is an important concept, it describes external data to be used as input/output to Cyclone workflows.
```

Comment:
There is no need to emphasize the importance of `Resource` concept, as other concepts are also important.

```suggestion
Resource describes external data to be used as input/output of Cyclone workflows.
```